pygyat/parser.py

```diff
@@ -73,6 +73,35 @@ def parse_glazes(filename):
     return glazes_with_suffixes
 
 
+def safe_substitute(value, deescaped_key, line):
```

Comment:
Nice work! Could you also use this function in [scripts/py2gyat](https://github.com/branyip/pygyat/blob/safe-substitutes/scripts/py2gyat) as well? The regex works pretty similarly as of right now so this fix should be easily applicable to py2gyat as well.

Looks good to merge after that!

Reply:
@shamith09 Added safe substitute to `py2gyat`, also fixed an existing bug in it related to the escape characters in the mapping.

Before:
<img width="662" alt="image" src="https://github.com/user-attachments/assets/cb8b7dab-8ced-45fa-99f6-a5a37660070c">

After:
<img width="448" alt="image" src="https://github.com/user-attachments/assets/a2df1f5c-f4bc-48f9-88de-0cc64d5c0ca0">